runtime/doc/eval.txt

```diff
@@ -5405,13 +5405,25 @@ server2client( {clientid}, {string})			*server2client()*
 			:echo server2client(expand("<client>"), "HELLO")
 <
 serverlist()					*serverlist()*
-		Return a list of available server names, one per line.
-		When there are no servers or the information is not available
-		an empty string is returned.  See also |clientserver|.
-		{only available when compiled with the |+clientserver| feature}
+		Return a list of available server names in a list.
+		When there are no servers an empty string is returned.
 		Example: >
 			:echo serverlist()
 <
+serverlisten([{address}])				*serverlisten()*
+		Opens a unix or tcp socket at {address} for clients to connect
+		to and returns {address}. If no address is given, it is
+		equivalent to >
+			:call serverlisten(tempname())
+< 		If `$NVIM_LISTEN_ADDRESS` is not set, it will be set to
```

Comment:
Maybe the language could use a little work, but "if |$NVIM_LISTEN_ADDRESS| is not set, it is set" is a contradictory statement. But no, it's not deferred.